SWIPs/swip-honey_to_money.md

```diff
@@ -9,29 +9,29 @@ Created: 31-07-2019
 ## Simple summary 
-Swarm needs a mechanism to allow fluctuation of the exchange rate between honey (Swarms internal accounting unit) and the currency used to pay. We propose an on-chain price oracle that returns an exchange rate + validUntil tuple.  
+Swarm needs a mechanism to allow fluctuation of the exchange rate between honey (Swarms internal accounting unit) and the currency used to pay. We propose an on-chain price oracle that returns an `(exchange rate,validUntil)` tuple.  
 
 ## Abstract 
 The ability to update prices is required in order to:
 * Facilitate experimentation with the absolute price of honey (e.g. start at a low price and gradually increase);
-* Off-set fluctuations of the currency used to pay;
-* Lower the price of Swarm in the long-run as the costs of operating a Swarm node decreases.
+* Offset fluctuations of the currency used to pay;
+* Lower the price of Swarm in the long run as the costs of operating a Swarm node decreases.
 
 We propose an on-chain price oracle that returns the current exchange rate when queried. The price-oracle will be initially managed by Swarm developers and stakeholders.
 
 This SWIP is part of a series of SWIPs (but can be implemented on it's own). To see the full picture, please refer to [swip-message_to_honey](./swip-message_to_honey.md), [swip-multiple](./swip-honey_to_money.md), [swip-multiple_payment_processing_support](./swip-multiple_payment_processing_support.md) and the diagram below:
 ![SWIP_Diagrams.svg](./../assets/swip-honey_to_money/SWIP_Diagrams.svg)
 
 ## Motivation
-Nodes keep track of the balances with each other nodes using an imaginary currency: honey. Since honey is not a currency in which nodes can settle their balances with each other - it is merely a unit of account - there is a requirement to facilitate converting honey to a currency. Furthermore, it is desirable that this conversion rate can fluctuate since:
+Nodes keep track of the balances with each other nodes using an internal reference unit: honey. Since honey is not a currency in which nodes can settle their balances with each other - it is merely a unit of account - there is a requirement to facilitate converting honey to a currency. Furthermore, it is desirable that this conversion rate can fluctuate since:
 
 * The Swarm developers plan to initially set the price of honey very low and progressively increase this price to the point where it is sufficiently high to incentivize nodes to offer their service to the network while being low enough not to scare away potential users of Swarm. 
 * We expect the price of the counterpart currency to fluctuate in value against the costs of operating a Swarm node. As the real benefits (i.e. the worth of your compensation) of operating a Swarm node shouldn't fluctuate, the honey price should fluctuate inversely against the fluctuations of currency. If this happens, the real benefit stays equal, regardless of price fluctuations of the currency in which compensation is paid. 
```

Comment:
What is the reasoning behind "the real benefits of operating a node shouldn't fluctuate"? What does it even mean?

Does a hosting provider not have to deal with the same problems? I don't think this is actually correct.

Reply:
I am sorry for using ununderstandable jargon here. We have nominal prices/return/benefits vs real prices/return/benefits. Nominal is always the amount mentioned (i.e. 100 dollar will always stay 100 dollar), but 100 dollar in real terms will be most likely less than 100 dollar in real terms (measured at the real price of today) in 10 years. 

This is what is meant with the above sentence. Real returns (i.e. the actual worth of their return) should not fluctuate, because if it does, it will be unattractive for professional nodes in the Swarm network to join, as it will be difficult to make predictions on their rate of return.  

If it helps you to frame: cryptocurrencies have such a mechanism as well: if the price of any proof-of-work cryptocurrency drops, mining for some miners will become unprofitable, mining rigs are shut down, the hash rate decreases, difficulty drops and from then onwards it will cost less (hashrate = electricity) to mine an equal amount of cryptocurrencies. 

What do you mean with 
> Does a hosting provider not have to deal with the same problems?

Reply:
I think we are again discussing the issue of value fluctuation, which I have expressed my opinion above.

It is the value of the cryptocurrency which fluctuates, not the operation of Swarm. The value of the cryptocurrency fluctuates against fiat. 

My hosting costs the same since 4 years now more or less, regardless of how much the USD fluctuates against the EUR. The profitability does not come from hedging against currency fluctuations but by the utility margins against the costs. IMHO.